docs/source/morpheus_quickstart_guide.md

```diff
@@ -199,7 +217,7 @@ $ helm fetch https://helm.ngc.nvidia.com/nvidia/morpheus/charts/morpheus-sdk-cli
 $ helm install --set ngc.apiKey="$API_KEY" \
```

Comment:
Could we move this command to a new section following the "Example Workflows" section, as Adam Wood recommended, add the section title to the table of contents?